### DISCUSSION TITLE
supportus/donate

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#56] [General]
# supportus/donate

支持GoFrame框架项目，通过微信、支付宝以及码云等渠道进行捐赠。捐赠者可以在支付备注中留下自己的GitHub或Gitee账号，帮助框架不断发展壮大。感谢所有支持者，推动开源社区的进步和发展。

https://pages.goframe.org/supportus/donate

**Comment by hailaz** [comment]
请大佬喝茶🍵
